src/server/pbsd_init.c

```diff
@@ -616,7 +616,8 @@ pbsd_init(int type)
 			/* Create and save default to DB*/
 			dflt_scheduler = sched_alloc(PBS_DFLT_SCHED_NAME);
 			set_sched_default(dflt_scheduler, 0);
-			(void)sched_save_db(dflt_scheduler, SVR_SAVE_NEW);
+			set_scheduler_flag(SCH_ATTRS_CONFIGURE, dflt_scheduler);
```

Comment:
I thought it would be safe to do this, just in case there were any race conditions before server went down the last time. Do you strongly feel that we should remove it?

Reply:
That's a pretty small race crash condition.  The server would have to crash between when the sched was saved to the DB and the next time through the main loop when the high priority scheduling cycle happened.  If you want this to happen, call the update attrs call in SCH_SCHEDULE_FIRST case in schedule().  This happens when the server first comes up.

Reply:
Ok, ya that might be better, thanks.